### DISCUSSION TITLE
prisma can implement `coursor.forEach(() => {})` like mongodb or based on stream?

### DISCUSSION THREAD

**Discussion by hncjygd** [prisma/prisma#16473] [Q&A]
I wonder if prisma can implement `coursor.forEach(() => {})` like mongodb?
Even like `cursor.stream().on("data", doc => console.log(doc))` based on stream。
I guess this is more javascriptic。

**Comment by nurul3101** [comment]
Hey @hncjygd 👋 

Can you please elaborate on what your use case is?
I am not able to fully understand what you are trying to achieve.

**Reply by vladi-stevanovic** [reply]
👋 Hi @hncjygd , did you perhaps have a chance to check Nurul's question? Let us know if you still need help here!

Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
